specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -253,10 +249,11 @@ these standard services are actually provided by Java SE.
 
 ==== HTTP
 
-The HTTP client-side API is defined by the
-_java.net_ package. The HTTP server-side API is defined by the Jakarta Servlet,
-Jakarta Server Pages, and Jakarta Server Faces interfaces and by the web services support that is an optional part
-of the Jakarta EE platform.
+The HTTP client-side API is defined by the _java.net_ package. The
+HTTP server-side API is defined and used by the Jakarta RESTful Web Services,
+Jakarta Servlet, Jakarta Server Pages, and Jakarta Server Faces
+interfaces and by the web services support that is an optional part of
+the Jakarta EE platform.
```

Comment:
```suggestion
HTTP server-side API is defined and used by the Jakarta RESTful Web Services,
Jakarta Servlet, Jakarta Server Pages, and Jakarta Server Faces
interfaces.
```

I contemplated removing `Jakarta RESTful Web Services` that you added since it doesn't actually have HTTP service-side APIs per se, but I assume you adding the `used by` was the reason you were able to add RESTful Web Services.

Reply:
You assume correctly. Neither Pages nor Faces define HTTP APIs. As you point out, REST also does not really define HTTP APIs in the same way as Servlet does.